frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -140,98 +144,120 @@ const Uploads = () => {
   };
 
   return (
-    <main className="prime-home">
-      <div className="grid-container">
-        <div className="grid-row">
-          <div className="prime-container">
-            <div className="usa-card__header">
-              <h2>COVID-19 CSV Uploads</h2>
-            </div>
-            {reportId && (
-              <div>
-                <div className="usa-alert usa-alert--success">
-                  <div className="usa-alert__body">
-                    <h4 className="usa-alert__heading">
-                      Success: File Accepted
-                    </h4>
-                    <p className="usa-alert__text">
-                      Your file has been successfully transmitted to the
-                      department of health
-                    </p>
-                  </div>
-                </div>
-                <div>
-                  <p className="text-normal text-base margin-bottom-0">
-                    Confirmation Code
+    <div className="grid-row">
+      <div className="prime-container card-container">
+        <div className="usa-card__header">
+          <h2>Upload your results</h2>
+        </div>
+        <div className="usa-card__body padding-y-2 maxw-prose">
+          <p>
+            Add and report results in bulk using a comma-separated values (CSV)
+            spreadsheet. To upload your spreadsheet:
+          </p>
+          <ul className="usa-list">
+            <li>
+              <a href="https://reportstream.cdc.gov/assets/csv/ReportStream-StandardCSV-ExampleData-20220509.csv">
+                Download the spreadsheet template
+              </a>{" "}
+              and open it in Excel or another spreadsheet application
+            </li>
+            <li>
+              Add your data to the relevant fields in the template, or if you’re
+              using your own spreadsheet, format column headers to match the
+              template
+            </li>
+            <li>
+              Make sure you include data for all fields marked “required” in row
+              2
+            </li>
+            <li>Once you’ve entered your data, delete all example data</li>
+            <li>Save your spreadsheet as a CSV file and upload it here</li>
+          </ul>
+          <p>
+            For more information about this process, visit our{" "}
+            <LinkWithQuery to="/results/upload/schema">
+              data template guide
+            </LinkWithQuery>{" "}
+            or{" "}
+            <LinkWithQuery to="/results/upload/guide">
+              upload guide.
+            </LinkWithQuery>
+          </p>
```

Comment:
```suggestion
          <p>
           Report results in bulk using a comma-separated values (CSV)
            spreadsheet. To upload your spreadsheet:
          </p>
          <ol className="usa-list">
            <li>
              <a href="https://reportstream.cdc.gov/assets/csv/ReportStream-StandardCSV-ExampleData-20220509.csv">
                Download the spreadsheet template
              </a>{" "}
            </li>
            <li>
              Format and add your data following the template and upload guide
            </li>
            <li>
              Save your spreadsheet as a CSV
            </li>
            <li>Upload your CSV here</li>
          </ol>
          <p>
            For more information about preparing your spreadsheet, visit our{" "}
            <LinkWithQuery to="/results/upload/schema">
              spreadsheet upload guide
            </LinkWithQuery>{" "}
          </p>
```

Reply:
@lauraykerr it looks like this change removes the link to the "upload guide" at `/results/upload/guide`. Now it only has the link to the CSV schema. Do we want to preserve the path to the guide here?

Reply:
@nathancrtr With design feedback, we decided to consolidate the upload guide and schema documentation (see suggestions for the new section in the schema file).